cicd/template.yml

```diff
@@ -231,8 +231,8 @@ Resources:
   TaskDefinition:
     Type: AWS::ECS::TaskDefinition
     Properties:
-      Cpu: "512"
-      Memory: "1024"
+      Cpu: "256"
```

Comment:
Originally thought low resources was causing frequent disconnects, this was not the case.